prime-router/metadata/hl7_mapping/ORU_R01/base/order.yml

```diff
@@ -38,25 +38,30 @@ elements:
     resourceIndex: entityIdIndex
     schema: entity-identifier
 
-  - name: response-flag
+  - name: order-response-flag
     resource: '%resource.extension(%`rsext-response-flag`)'
     condition: '%resource.exists()'
     value: [ '%resource.value.coding[0].code' ]
     hl7Spec: [ '%{hl7Order}-6' ]
 
-  - name: transaction-time
+  - name: order-transaction-time
     resource: '%resource.basedOn.resolve()'
     value: [ '%resource.authoredOn' ]
     hl7Spec: [ '%{hl7Order}-9' ]
 
-  - name: ordering-provider
+  - name: order-entered-by
+    condition: false
+    value: [ '' ] # Don't have data yet
```

Comment:
This needs to be `''''''` instead of just `''`. Its being evaluated as a json path instead of an empty string and throwing an error.